### PR TITLE
elasticsearch and kibana: removed minor version from aliases

### PR DESCRIPTION
**Pull request by ankane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Removes minor versions from Elasticsearch and Kibana aliases.

### CODE REVIEW

**Comment by ankane** [comment]
Failures due to `New formulae in homebrew/core should not have a `bottle do` block`. Should these be removed?

**Comment by jonchang** [comment]
Yeah, because the formula name changed we'll need new bottles. Can you also add entries to formula_renames.json? Thanks!

**Comment by jonchang** [comment]
Also, I'm assuming this is being done because upstream only maintains a few major versions, and not e.g. major+minor versions?

**Comment by ankane** [comment]
Yeah, that (support policy: https://www.elastic.co/support/eol) and the (probably related) fact that Homebrew doesn't maintain separate formula for all the minor versions either.

File: Formula/elasticsearch@2.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-class ElasticsearchAT24 < Formula
+class ElasticsearchAT2 < Formula
```
Comment (SMillerDev) [inline Formula/elasticsearch@2.rb:1]: This should just be removed

File: Formula/elasticsearch@5.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-class ElasticsearchAT56 < Formula
+class ElasticsearchAT5 < Formula
```
Comment (SMillerDev) [inline Formula/elasticsearch@5.rb:1]: Should be removed

**Review by SMillerDev — APPROVED** [review]
Thanks @ankane  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**